apps/standalone/src/app/components/AppLayout/AppToolbar.tsx

```diff
@@ -1,21 +1,26 @@
 import {
+  Alert,
   Avatar,
   Button,
   Dropdown,
   DropdownItem,
   DropdownList,
   MenuToggle,
   MenuToggleElement,
+  Modal,
   Toolbar,
   ToolbarContent,
   ToolbarItem,
 } from '@patternfly/react-core';
 import * as React from 'react';
-import { useAuth } from '../../hooks/useAuth';
 
-import './AppToolbar.css';
 import UserPreferencesModal from '@flightctl/ui-components/src/components/UserPreferences/UserPreferencesModal';
 import { useTranslation } from '@flightctl/ui-components/src/hooks/useTranslation';
+import { getErrorMessage } from '@flightctl/ui-components/src/utils/error';
+import { AuthContext } from '../../context/AuthContext';
+import { logout } from '../../utils/apiCalls';
+
+import './AppToolbar.css';
```

Comment:
Moving the CSS import to the bottom helped my OCD :)

Reply:
mine too! :)